pep-0668.rst

```diff
@@ -0,0 +1,1099 @@
+    itself, software written in Python, and software written in other
+    languages. That is, this is the sense used in phrases such as
+    "Linux distro" or "Berkeley Software Distribution."
+
+    A distro can be an operating system (OS) of its own, such as
+    Debian, Fedora, or FreeBSD. It can also be an overlay distribution
+    that installs on top of an existing OS, such as Homebrew or
+    MacPorts.
+
+    This document uses the short term "distro," because the term
+    "distribution" has another meaning in Python packaging contexts: a
+    source or binary distribution package of a single piece of Python
+    language software, that is, in the sense of
+    ``setuptools.dist.Distribution`` or "sdist". To avoid confusion,
+    this document does not use the plain term "distribution" at all.
+    In the Python packaging sense, it uses the full phrase
+    "distribution package" or just "package" (see below).
+
+    The provider of a distro - the team or company that collects and
+    publishes the software and makes any needed modifications - is its
+    **distributor**.
+package
+    A unit of software that can be installed and used within Python.
+    That is, this refers to what Python-specific packaging tools tend
+    to call a "`distribution package`_" or simply a "distribution";
+    the colloquial abbreviation "package" is used in the sense of the
+    Python Package Index.
+
+    .. _`distribution package`: https://packaging.python.org/glossary/#term-Distribution-Package
+
+    This document does not use "package" in the sense of an importable
+    name that contains Python modules, though in many cases, a
+    distribution package consists of a single importable package of
+    the same name.
+
+    This document generally does not use the term "package" to refer
+    to units of installation by a distro's package manager (such as
+    ``.deb`` or ``.rpm`` files). When needed, it uses phrasing such as
+    "a distro's package." (Again, in many cases, a Python package is
+    shipped inside a distro's package named something like ``python-``
+    plus the Python package name.)
+Python-specific package manager
+    A tool for installing, upgrading, and/or removing Python packages
+    in a manner that conforms to Python packaging standards (such as
+    PEP 376 [#PEP-376]_ and PEP 427 [#PEP-427]_). The most popular
+    Python-specific package manager is pip [#pip]_; other examples
+    include the old Easy Install command [#easy-install]_ as well as
+    direct usage of a ``setup.py`` command.
+
+    (Conda [#conda]_ is a bit of a special case, as the ``conda``
+    command can install much more than just Python packages, making it
+    more like a distro package manager in some senses. Since the
+    ``conda`` command generally only operates on Conda-created
+    environments, most of the concerns in this document do not apply
+    to ``conda`` when acting as a Python-specific package manager.)
+distro package manager
```

Comment:
FWIW Conda supports installing Python packages via the pip CLI in its environment.yml files: https://github.com/conda/conda/blob/master/tests/conda_env/support/advanced-pip/environment.yml

So making sure we know where packages came from will come in handy in the future.

Plus, there is a off-by-default feature to take Python packages into consideration in the dependency solver stage: https://docs.conda.io/projects/conda/en/latest/user-guide/configuration/pip-interoperability.html It's likely that this will be enabled by default in the future to improve interoperability since it's something Conda users have requested heavily.